moduleroot/.gitlab-ci.yml.erb

```diff
@@ -91,6 +91,22 @@ before_script:
   allow_failure: true
 <%         end -%>
 <%       end -%>
+<%       if options['except'] -%>
+<%         if options['except'].has_key?(check) -%>
+  except:
+<%           options['except'][check].each do |ref| -%>
+    - <%= ref %>
+<%           end -%>
+<%         end -%>
+<%       end -%>
+<%       if options['only'] -%>
+<%         if options['only'].has_key?(check) -%>
+  only:
+<%           options['only'][check].each do |ref| -%>
```

Comment:
Same as above. (This might need to guard against `options['only'][check]` being `nil` since `has_key?` won't protect from the value being `nil`.)

Reply:
```suggestion
<%           options['only'][check] && options['only'][check].each do |ref| -%>
```